File: 프로그래머스/0/181943. 문자열 겹쳐쓰기/문자열 겹쳐쓰기.java
class Solution {
    public String solution(String my_string, String overwrite_string, int s) {
        String answer = "";
        int len1 = my_string.length();
        int len2 = overwrite_string.length();
        String str1 = my_string.substring(0,s);
        String str2 = overwrite_string + my_string.substring(s+len2,len1) ;
        answer = str1 +str2;
        
        return answer;
    }
}

File: 프로그래머스/0/181943. 문자열 겹쳐쓰기/문자열 겹쳐쓰기.py
def solution(my_string, overwrite_string, s):
    answer = ''
    len1 = len(my_string)
    len2 = len(overwrite_string)
    str1 = my_string[:s]
    str2 = overwrite_string+my_string[s+len2:] 
    answer = str1 + str2
    return answer